celi_framework/main.py

```diff
@@ -146,6 +227,20 @@ def instantiate_with_argparse_args(args: argparse.Namespace, cls: Type):
 
 
 if __name__ == "__main__":
+    tool_config_path = get_default_config_path()
+    print(f"Attempting to use config at: {tool_config_path}")
```

Comment:
logger. is generally way nicer than 'print' for a few reasons.
You can control logging levels with fine granuality.
In production applications you can redirect logging to other places besides stderr.
You can get line number and file information, which you can't with print.  When code bases get bigger trying to track down a random print statement can be really hard, especiailly when the string is generated in code.
I'd recommend logger.debug here.